projects/community-payback-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/AppointmentsService.kt

```diff
@@ -119,58 +119,55 @@ class AppointmentsService(
         )
     }
 
+    @Transactional
     fun updateAppointmentOutcome(
         projectCode: String,
         appointmentId: Long,
         appointmentOutcome: AppointmentOutcomeRequest
```

Comment:
Minor thing - could we rename this parameter to `request`?  I think it would make the rest of the code a bit clearer.